task_sdk/src/airflow/sdk/definitions/_internal/abstractoperator.py

```diff
@@ -136,6 +138,39 @@ class AbstractOperator(Templater, DAGNode):
         )
     )
 
+    @cached_property
+    def operator_extra_link_dict(self) -> dict[str, Any]:
```

Comment:
Could/should this be `-> dict[str, BaseOperatorLink]` instead?

Reply:
Yep it should be. I just ported it over the same way, so missed it.
Handled in latest commit